lib/cli.ex

```diff
@@ -0,0 +1,196 @@
+defmodule Gradient.CLI do
+  @moduledoc """
+  Gradient CLI module responsible for
+  accepting shell params and starting gradualizer.
+
+  ## Command-line options
+
+    * `--no-compile` - do not compile even if needed
+    * `--no-ex-check` - do not perform checks specyfic for Elixir
+      (from ElixirChecker module)
+    * `--no-gradualizer-check` - do not perform the Gradualizer checks
+    * `--no-specify` - do not specify missing lines in AST what can
+      result in less precise error messages
+    * `--code-path` -  provide a path to the .ex file containing code for analyzed .beam
```

Comment:
Can we also rename this option to `source-path`? Using _code path_ here is somewhat misleading, as it might suggest we're thinking of [the Erlang code path](https://www.erlang.org/doc/man/code.html#code-path), while it's not the case. It's even more misleading in the light of introducing `path-add`, which actually appends paths to the Erlang code path.

Reply:
Sorry, I missed this one. Yes, please change it everywhere: https://github.com/esl/gradient/pull/121#discussion_r958139448.

Reply:
> we would need to change it in Gradualizer itself

BTW, I don't see why this would be the case. This option is defined in Gradient and consumed in Gradient, it's not passed to Gradualizer at all.

Reply:
done